src/util/document.js

```diff
@@ -0,0 +1,29 @@
+import fetch from 'node-fetch';
```

Comment:
I'm not sure this file is necessary. Also node-fetch may be possible issue if used in browser?

Reply:
> I'm not sure this file is necessary.

Ok I see you have a client-side module.

Reply:
I'm pretty sure `fetch()` should just be available globally on the server.  Doing separate `import`s each time fetch is used on the server could make for more refactoring work later if we ever have to swap out the polyfill.

Reply:
It gives an error by telling that fetch is not defined in server side if I do not import fetch. I tried that it still works in client side as well when I import the node-fetch.